src/api/java/baritone/api/Settings.java

```diff
@@ -72,7 +72,7 @@
     /**
      * If this setting is on, no auto tool will occur at all, not at calculation time nor execution time
```

Comment:
```suggestion
     * If this setting is off, no auto tool will occur at all, not at calculation time nor execution time
```

Reply:
im unsure of any other places where this might change.

Reply:
Did a grep for 'autoTool' over the whole repo and didn't find any more.
Sorry for not accepting your suggestion, but I wanted to have the shorter wording.

Reply:
LGTM